Makefile

```diff
@@ -5,7 +5,11 @@ IMG ?= ${REGISTRY}/csi-baremetal-operator:${TAG}
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true"
-
+# Variables for CRDs generation, require csi-baremetal repository
+CSI_BAREMETAL_DRIVER_DIR=../csi-baremetal
+CSI_CHART_CRDS_PATH=charts/csi-baremetal-operator/crds
+# TODO install controller-tool >= 0.5.0 implement version freezy in go.mod
+CONTROLLER_GEN_BIN=./bin/controller-gen
```

Comment:
Bin to `.gitignore`?

Reply:
Plan to add a separate target for build `controller-gen` to bin folder.